src/styles/index.scss

```diff
@@ -1,3 +1,2 @@
-body {
-  margin: 0;
-}
+@import url(../blocks/card.scss);
+@import url(../blocks/stars.scss);
```

Comment:
Make sure the paths to the SCSS files are correct and the files exist in those locations.